src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  flex-direction: column;
+
+  box-sizing: border-box;
+  width: $card-width;
```

Comment:
It is better not to use fixed values, bc if there is not enough width for the card - the card adapts better

max-width: 200px;
width: 100%;